### ISSUE TITLE
Documentation: Versioning page description for "+" releases of WET?

### ISSUE THREAD

**Issue by EricDunsworth** [wet-boew/wet-boew#6306]
@LaurentGoderre Would you see any value in documenting what differentiates + releases from regular maintenance releases on the versioning page (http://wet-boew.github.io/wet-boew/docs/versions/version-info-en.html)?

**Comment by EricDunsworth** [comment]
IMO there'd still be value in documenting the meaning of + releases in WET's versioning page. If all other kinds of version types are documented, why not document + too?

I should also note that the semver.org link doesn't clearly describe what its take on + releases are (I assume it's patches) and nothing in WET's documentation states that WET uses semantic versioning.

**Comment by hsrudnicki** [comment]
@EricDunsworth @LaurentGoderre : Are you ready to close this issue?

**Comment by EricDunsworth** [comment]
@hsrudnicki The documentation discussed in this issue hasn't been added yet.

@LaurentGoderre Do you expect there to be any more "+" releases of WET in the future? If not, I wouldn't mind closing this issue.

The reason I had opened this issue instead of sending a PR in the first place was because I don't know what distinguishes a "+" release from a maintenance release (and still don't). I figured you'd be in the best position to document it since you launch the releases.

**Comment by LaurentGoderre** [comment]
Ideally it would never happen again but that is unlikely. Basically, you can't revert a version so if something critical is discovered after the release, it allows to add a quick fix while still sticking with our monthly releases.

**Comment by hsrudnicki** [comment]
Made "Going Stale" and closed this issue due to not continued discussion and a dilemma or if you strongly think otherwise, then please update this issue's status again, thank you

**Comment by EricDunsworth** [comment]
@hsrudnicki I should note that this issue is still technically valid. If there's an expectation that "+" releases of WET might occur again in the future, it would make sense to document what kinds of changes can be expected from them in the versioning page. IMO, just because this issue is old doesn't make it less valid than when it was first opened.

I can't do anything to address it since it's not me that launches the releases. I'm not in the know about what circumstances can lead to "+" releases of WET nor what kinds of changes they can contain. @MaximMartel and/or @andre-gh are probably in the best position to properly resolve this issue.

Btw just in case no one knows what this issue is talking about anymore, [WET 4.0.8](http://wet-boew.github.io/wet-boew/docs/versions/v4.0/v4.0.8-en.html) was released on November 18, 2014. [WET 4.0.8+1](http://wet-boew.github.io/wet-boew/docs/versions/v4.0/v4.0.8+1-en.html) and [4.0.8+2](http://wet-boew.github.io/wet-boew/docs/versions/v4.0/v4.0.8+2-en.html) came out a couple of days later. I opened this issue because the versioning page has no documentation about release timelines, changes, etc that can be expected out of "+" releases.
